app/models/manageiq/providers/openstack/cloud_manager/template.rb

```diff
@@ -99,6 +99,8 @@ def self.create_image(ext_management_system, create_options)
 
   def raw_update_image(options)
     ext_management_system.with_provider_connection(:service => 'Image') do |service|
+      image_attrs = self.attributes
```

Comment:
This seems to fix issue with miq_queue keys in attributes, but to have it fully correct, it is needed check attributes against Fog Image instead of MIQ Template.

```self.attributes``` are attributes of Template class, ```service.images.find_by_id(ems_ref)``` returns Fog object - https://github.com/fog/fog-openstack/blob/master/lib/fog/image/openstack/v2/models/image.rb its attributes should be checked.

Reply:
Updated PR